aten/src/ATen/Context.h

```diff
@@ -142,22 +142,22 @@ CAFFE2_API Allocator* getCPUAllocator();
 
 static inline DeprecatedTypeProperties& getNonVariableDeprecatedTypeProperties(Backend p, ScalarType s) {
   return globalDeprecatedTypePropertiesRegistry().getDeprecatedTypeProperties(
-      p, s, /*is_variable*/false);
+      p, s);
```

Comment:
umm...this seems super misleading to leave this function as is.

Reply:
Trying to keep this PR on the smaller side :)

Reply:
you could put up a pre-req PR.

Reply:
#29203